src/test/scala/itc/Test.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic
+
+import basic.enum._
+import basic.itc.ItcImpl
+import basic.misc._
+import cats.effect._
+import cats.tests._
+import gem.enum._
+import gem.math.Wavelength
+import org.scalatest.Assertion
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+
+class ItcTest extends CatsSuite {
+
+  implicit val contextShift: ContextShift[IO] =
+    IO.contextShift(ExecutionContext.global)
+
+  def targetProfile(magnitude: Double) =
+    TargetProfile(
+      spatialProfile       = SpatialProfile.UniformSource,
+      spectralDistribution = SpectralDistribution.Library(Right(NonStellarLibrarySpectrum.Neptune)),
+      magnitude            = magnitude,
+      magnitudeSystem      = MagnitudeSystem.Vega,
+      magnitudeBand        = MagnitudeBand.R,
+      redshift             = Redshift(0.0)
+    )
+
+  val observingMode =
+    ObservingMode.Spectroscopy.GmosNorth(
+      λ         = Wavelength.fromNanometers.unsafeGet(755),
+      disperser = GmosNorthDisperser.B600_G5307,
+      fpu       = GmosNorthFpu.LongSlit_0_25,
+      filter    = Some(GmosNorthFilter.OG515)
+    )
+
+  def withItc(f: Itc[IO] => IO[Assertion]): Assertion =
+    ItcImpl.forHeroku[IO].use(f).unsafeRunSync
```

Comment:
I'm uneasy about seeing unit tests that depend on external resources. We do need tests but it would be nice to have the option of skipping anything that requires a network connection and an external service to be running.

Reply:
Yeah this is temporary. I'll add that "slow test" flag.